pkg/pb/metainfo.proto

```diff
@@ -103,4 +104,15 @@ message ListSegmentsResponse {
       
     repeated Item items = 1;
     bool more = 2;
+}
+
+message ValueAttributionRequest{
+    bytes bucket_id = 1;
+    bytes user_id = 2;
+    bytes partner_id = 3;
+    SegmentInfoRequest segment_info_request = 4;
+}
+
+message ValueAttributionResponse {
+    bool value_attribution = 1; 
```

Comment:
Why `value_attribution` is needed?

Reply:
protobuf needed some response. so I added this bool . I know error value nil and not nil can be used in stead of it. Removed